lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1286,7 +1296,6 @@ function dosomething_user_set_fields(&$edit, $fields) {
     }
 
     switch ($key) {
-      case 'country':
```

Comment:
nope! that was from debugging a different issue I was running into

(tho it does beg the question why this was here to begin with, there isnt even a break statement)